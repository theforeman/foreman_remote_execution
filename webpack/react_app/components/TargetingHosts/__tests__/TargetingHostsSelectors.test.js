import { testSelectorsSnapshotWithFixtures } from '@theforeman/test';

import {
  selectItems,
  selectAutoRefresh,
  selectApiStatus,
  selectTotalHosts,
  selectIntervalExists,
} from '../TargetingHostsSelectors';

const state = {
  hosts: [],
  autoRefresh: 'true',
  total_hosts: 0,
};

const fixtures = {
  'should return hosts': () => selectItems(state),
  'should return autoRefresh': () => selectAutoRefresh(state),
  'should return apiStatus': () => selectApiStatus(state),
  'should return totalHosts': () => selectTotalHosts(state),
  'should return intervalExists': () => selectIntervalExists(state),
};

describe('TargetingHostsSelectors', () =>
  testSelectorsSnapshotWithFixtures(fixtures));
