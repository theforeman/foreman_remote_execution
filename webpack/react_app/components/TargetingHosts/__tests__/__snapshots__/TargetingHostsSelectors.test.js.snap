// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TargetingHostsSelectors should return apiStatus 1`] = `"RESOLVED"`;

exports[`TargetingHostsSelectors should return autoRefresh 1`] = `"true"`;

exports[`TargetingHostsSelectors should return hosts 1`] = `Array []`;

exports[`TargetingHostsSelectors should return intervalExists 1`] = `false`;

exports[`TargetingHostsSelectors should return totalHosts 1`] = `0`;
